packages/drivers/odsp-driver/src/odspDriverUrlResolverForShareLink.ts

```diff
@@ -168,6 +170,11 @@ export class OdspDriverUrlResolverForShareLink implements IUrlResolver {
 			odspResolvedUrl.shareLinkInfo = Object.assign(odspResolvedUrl.shareLinkInfo ?? {}, {
 				sharingLinkToRedeem: this.removeNavParam(request.url),
 			});
+			if (isNonDurableRedeem) {
+				odspResolvedUrl.shareLinkInfo = Object.assign(odspResolvedUrl.shareLinkInfo ?? {}, {
```

Comment:
instead of doing it again, you can just do this above in the same assignment and set to true/false.